src/main/java/io/seqera/migtool/Helper.java

```diff
@@ -20,27 +25,25 @@
  */
 public class Helper {
 
-    public static List<String> getResourceFiles(String path) {
+    public static SortedSet<String> getResourceFiles(String path) {
         if( path==null )
```

Comment:
Nice, tho I don't the SortedSet is needed since they get sorted when applying  the migration 

https://github.com/seqeralabs/migtool/blob/8833fe735f1401324f7b3074de3fe58dd26d8539/src/main/java/io/seqera/migtool/MigTool.java#L204

Reply:
Cool. I applied the `SortedSet` because, strangely, the resources were being retrieved twice when reading from the JAR:
https://github.com/seqeralabs/migtool/pull/1/files#diff-9be0847ff1939aaeea9adfb6726b8bb00b36cdc467efa6e1d576e11114f9e2b9R168-R173
This way both properties (order and uniqueness) are guaranteed. `Collections.sort()` can only be applied on `List`s though.

Reply:
Fair enough